App.tsx

```diff
@@ -29,12 +29,11 @@ export default function App(): React.ReactElement | null {
     if (!isLoadingComplete) {
         return null;
     }
+
     return (
-        <Provider store={store}>
```

Comment:
Store provider not needed ?

Reply:
Not yet, it was crashing the app, we'll add it when we'll have the user reducer